bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerRecoveryOp.java

```diff
@@ -110,7 +116,7 @@ private void doRecoveryRead() {
         if (!callbackDone.get()) {
             entryToRead++;
             try {
-                new RecoveryReadOp(lh, lh.bk.scheduler, entryToRead, entryToRead, this, null).initiate();
+                new RecoveryReadOp(lh, lh.bk.scheduler, entryToRead, entryToRead, this, null).parallelRead(parallelRead).initiate();
```

Comment:
You are assuming that if parallelRead is enabled, it is used for recoveryRead. I bet majority is fine with slow recovery but use faster/low latency parallel reads for application demanding workloads. So we need another configuration variable to allow parallel reads (if enabled) to use during recovery.